src/orange/orangeinp/IntersectRegion.hh

```diff
@@ -84,6 +86,49 @@ class Box final : public IntersectRegionInterface
     Real3 hw_;
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * An involute centered on the origin.
+ *
+ * \note Be aware there's also an involute *surface* at orange/surf/Involute.hh 
+ * in a different namespace.
+ */
+class Involute final : public IntersectRegionInterface
+{
+  public:
+    //! Enum defining chirality of involute
+    using Sign = celeritas::detail::InvoluteSolver::Sign;
+
+
+    // Construct with radius
+    explicit Involute(Real3 const& radii, Real2 const& displacement, 
+                      Sign sign, real_type halfheight);
+
+    // Build surfaces
+    void build(IntersectSurfaceBuilder&) const final;
+
+    // Output to JSON
+    void output(JsonPimpl*) const final;
+
+    //// ACCESSORS ////
+
+    //! Radius
+    Real3 radii() const { return radii_; }
+    Real2 a() const { return a_; }
```

Comment:
`a` isn't a very descriptive name (I think we should also rename it on the surface).  Can you rename to `displacement_angle` or something?